Platform/Intel/SimicsOpenBoardPkg/Library/MmPlatformHobProducerLib/MmPlatformHobProducerLib.inf

```diff
@@ -0,0 +1,39 @@
+## @file
+# Instance of platform specific HOB Producer Library for Standalone MM Core.
+#
+# Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010017
+  BASE_NAME                      = MmPlatformHobProducerLib
+  FILE_GUID                      = 98700A68-CB99-4B2A-97DC-F0A0C8755EE7
+  MODULE_TYPE                    = PEIM
+  VERSION_STRING                 = 1.0
+  LIBRARY_CLASS                  = MmPlatformHobProducerLib
+
+#
+#  VALID_ARCHITECTURES           = IA32 X64 ARM AARCH64
+#
+
+[Sources]
+  MmPlatformHobProducerLib.c
+
+[Packages]
+  MdePkg/MdePkg.dec
+  UefiCpuPkg/UefiCpuPkg.dec
+  SimicsOpenBoardPkg/OpenBoardPkg.dec

```

Comment:
Add StandaloneMmPkg/StandaloneMmPkg.dec since you need include the #include <Library/MmPlatformHobProducerLib.h>

Reply:
Updated